src/redux/actions.js

```diff
@@ -0,0 +1,9 @@
+import * as types from './actionTypes';
+import { isDisplayable } from '../pressButtonLogic';
+
+export function getType(value) {
```

Comment:
This is not a good name. And here you shouldn't have logic, at least not in this case. You need only a simple action that gets dispatched when a button is pressed. So this one should look like this: 
```
export pressButton = value => ({
   type: types.PRESS_BUTTON,
});
```